client/my-sites/posts/post-controls.jsx

```diff
@@ -64,7 +64,7 @@ const getAvailableControls = props => {
 
 		controls.main.push( {
 			className: 'stats',
-			href: `/stats/post/${ post.ID }/${ site.slug }`,
+			href: `/stats/post/${ post.ID }/${ siteSlug }`,
```

Comment:
Yeah, either that, or it'll be blocked by #13094. Tho I could at least break out the chances as `Post` component level and below into a separate PR...